### PR TITLE
Nightwatch V2 typings

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## nightwatch ([<kbd>unpkg</kbd>](https://unpkg.com/browse/nightwatch@latest/))
was missing the following properties:
1. createClient
1. client
1. cli
1. runTests
1. CliRunner

<details>
<summary>as well as these 2 other properties...</summary>
<p>initClient, runner</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e6d6008ca40225e5ad5b288c389acee53638fba0
</p>

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## nightwatch ([<kbd>unpkg</kbd>](https://unpkg.com/browse/nightwatch@latest/))
was missing the following properties:
1. createClient
1. client
1. cli
1. runTests
1. CliRunner

<details>
<summary>as well as these 2 other properties...</summary>
<p>initClient, runner</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 48a7aab50331c9947418b31e668c3e0a60d56a61
</p>

**Comment by wallzero** [comment]
Nice work!

The only part I am concerned about is you adding maintainers besides yourself. I'd rather you include yourself and they add themselves in their own PR. Then again when you become a maintainer you can ignore this; but I'd rather know individuals consented before becoming maintainers.

**Comment by wallzero** [comment]
@AutomatedTester @gravityvi @swrdfish @harshit-bs

Or you could each comment here and give consent.

File: types/nightwatch/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,19 @@
-// Type definitions for nightwatch 1.3
+// Type definitions for nightwatch 2.0
 // Project: http://nightwatchjs.org
 // Definitions by: Rahul Kavalapara <https://github.com/rkavalap>
 //                 Connor Schlesiger <https://github.com/schlesiger>
 //                 Clayton Astrom <https://github.com/ClaytonAstrom>
 //                 Lukas Beranek <https://github.com/lloiser>
+//                 Vaibhav Singh <https://github.com/vaibhavsingh97>
```
Comment (schlesiger) [inline types/nightwatch/index.d.ts:7]: I need consent from the other users.

**Review by schlesiger — CHANGES_REQUESTED** [review]

File: types/nightwatch/index.d.ts (hunk 1 above)
Comment (AutomatedTester) [reply types/nightwatch/index.d.ts:7]: I consent

Comment (AutomatedTester) [reply types/nightwatch/index.d.ts:7]: The new names are part of the @Nightwatchjs team at @BrowserStack. I will get them to 👍. I consent

**Review by swrdfish — APPROVED** [review]

**Review by harshit-bs — APPROVED** [review]

**Review by schlesiger — APPROVED** [review]

**Comment by schlesiger** [comment]
Ready to merge

**Review by schlesiger — COMMENTED** [review]
👍️

**Pull request merged**